src/ngx-gallery-options.model.ts

```diff
@@ -3,6 +3,14 @@ import { NgxGalleryImageSize } from './ngx-gallery-image-size.model';
 import { NgxGalleryLayout } from './ngx-gallery-layout.model';
 import { NgxGalleryOrder } from './ngx-gallery-order.model';
 
+export interface INgxGalleryAction {
+    icon: string;
+    disabled?: boolean;
+    titleText?: string;
+
+    onClick: (event: Event) => void;
+}
+
```

Comment:
Could you move `INgxGalleryAction` from `ngx-gallery-options.model.ts` to `ngx-gallery-action.model.ts` and export class `NgxGalleryAction` - it will be consistent with others